homeassistant/components/simplisafe/button.py

```diff
@@ -0,0 +1,93 @@
+"""Buttons for the SimpliSafe integration."""
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+
+from simplipy.errors import SimplipyError
+from simplipy.system import System
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import SimpliSafe, SimpliSafeEntity
+from .const import DOMAIN
+from .typing import SystemType
+
+
+@dataclass
+class SimpliSafeButtonDescriptionMixin:
+    """Define an entity description mixin for SimpliSafe buttons."""
+
+    push_action: Callable[[System], Awaitable]
+
+
+@dataclass
+class SimpliSafeButtonDescription(
+    ButtonEntityDescription, SimpliSafeButtonDescriptionMixin
+):
+    """Describe a SimpliSafe button description."""
+
+
+BUTTON_KIND_CLEAR_NOTIFICATIONS = "clear_notifications"
```

Comment:
This const is only used in one place, and it's right next to the button `name` property which is also a const.  If this isn't special then can we inline it?

This isn't consistent across the rest of the codebase, so I can't say that one way is right or wrong, but there are plenty that don't use defined consts for the keys.  So it's just my preference that unless there's a good reason to use the const, I say get rid of it.

Reply:
Yup, that's fine, I'm just asking if the `name` property should be also be pulled out as a const too.

Reply:
The common pattern is to use constants for keys (since those will be referenced elsewhere) and not for the names unless that specific string is needed elsewhere.

Reply:
@bachya - Sounds good to me!